### ISSUE TITLE
Define the deme "size". Haploid, or diploid?

### ISSUE THREAD

**Issue by grahamgower** [popsim-consortium/demes-spec#72]
I fear we might have overlooked this fundamentally important decision.

**Comment by jeromekelleher** [comment]
I think it should number of individuals - the ploidy of these individuals is extrinsic to the demographic model specification.

Very good and important question, though!

**Comment by apragsdale** [comment]
Agreed - this should be the number of individuals. Will be important to clarify in the docs and paper.

**Comment by apragsdale** [comment]
Also brings up a question - should we have `ploidy` as a top-level field in the specification? The default I assume would be 2, but users might wish to specify models for haploid, tetraploid, whatever-ploid species.

**Comment by jeromekelleher** [comment]
Is it necessary for specifying the Demography, or is an extrinsic simulation parameter? (We have to draw the line somewhere)

**Comment by apragsdale** [comment]
Yeah you're right, we don't want it to balloon too much. So then are we agnostic about matters of ploidy and those sorts of genome biology, or do we specifically have diploid models in mind (e.g. when talking about selfing rates)

**Comment by jeromekelleher** [comment]
We're agnostic as far as I'm concerned, but I'm not into the details of selfing etc.

**Comment by grahamgower** [comment]
I'm inclined to think we might need a ploidy field too. Not having a ploidy field means a simulator can't simulate yaml files in a general way. The simulator will probably need to do different things for different ploidies, and there's no way to know that ploidy to use (without user intervention). Presumably folks will have a ploidy assumption for certain organisms (e.g. in stdpopsim), so the ploidy information will be stored *somewhere* (even if it's just assumed from the filename because it contains the organism name).

**Comment by jeromekelleher** [comment]
I don't follow the logic - how is ploidy different to recombination rate, or genome length, or selection coefficients,...? Demes is never meant to be a complete description of a simulation, just of the *demography*. (Which again makes me wonder why we're putting cloning and selfing in...)

**Comment by grahamgower** [comment]
To me, ploidy seems different to recombination rate and genome length, because a user can modify those for a given simulation and the demography still makes sense. But the demography only applies to one specific ploidy.

**Comment by apragsdale** [comment]
I agree with Graham. They seem different to me, as selfing probability and ploidy are parameters about individuals (how many genome copies they carry and how they mate on the individual level) vs parameters about the genome copies themselves, which as pointed out can vary quite a bit between simulations without changing the meaning of the demography (and life history?) model. Is that a useful way to think about it?

**Comment by grahamgower** [comment]
I think this ship has sailed, and we're not adding a `ploidy` field. But we do still need some statement that the deme sizes correspond to individuals (with whatever ploidy), rather than the number haploid chromosomes.
